.github/workflows/selfdrive_tests.yaml

```diff
@@ -344,7 +344,6 @@ jobs:
       - uses: actions/checkout@v4
         with:
           submodules: true
-          ref: ${{ github.event.pull_request.base.ref }}
```

Comment:
why was this removed? it's needed for this job

Reply:
I removed it earlier because I set --warn=all explicitly in the build openpilot step of that job and it didn't work since it would pull the Sconsctruct from the base.ref (Sconstruct of master), which didn't work for explicit warn=all setting. But I can add it back now, and it should work normally. doing it now.